tests/pjit_test.py

```diff
@@ -820,15 +903,15 @@ def testAxisResourcesMismatch(self):
       pjit(lambda x, y: x, p, p)(x, x)  # Error, but make sure we hint at tupling
     # TODO(apaszke): Disable implicit list casts and enable this
     # error = re.escape(
-        # r"pjit in_axis_resources specification must be a tree prefix of the "
-        # r"corresponding value, got specification (None, None, None) for value "
-        # r"tree PyTreeDef(([*, *, *],)). Note that pjit in_axis_resources that "
-        # r"are non-trivial pytrees should always be wrapped in a tuple representing "
-        # r"the argument list. In particular, you're passing in a single argument "
-        # r"which means that pjit in_axis_resources might need to be wrapped in a "
-        # r"singleton tuple.")
+    # r"pjit in_axis_resources specification must be a tree prefix of the "
+    # r"corresponding value, got specification (None, None, None) for value "
+    # r"tree PyTreeDef(([*, *, *],)). Note that pjit in_axis_resources that "
+    # r"are non-trivial pytrees should always be wrapped in a tuple representing "
+    # r"the argument list. In particular, you're passing in a single argument "
+    # r"which means that pjit in_axis_resources might need to be wrapped in a "
+    # r"singleton tuple.")
```

Comment:
Why did this formatting change?

Reply:
I suspect cider is automatically doing this. I let it go since it kept happening.